### DISCUSSION TITLE
Automatically Positioning + Resizing Upon Application Launch

### DISCUSSION THREAD

**Discussion by vexaran** [rxhanson/RectanglePro-Community#630] [Q&A] [answered]
Hello,

I was wondering if Rectangle Pro has the ability to automatically apply a custom window position and size to a specific application upon launch. Currently, I open an application and use the maximize keyboard shortcut, but having to do this repeatedly can be a bit tedious. If this feature is not available, is it something that could be considered for a future update?

Thank you!

**Comment by rxhanson** [comment, marked as answer]
Here's an example of doing this with Safari windows, but you can switch it to `Global` if you wanted this for every window from all apps:
<img width="1066" height="876" alt="image" src="https://github.com/user-attachments/assets/763af073-4a83-4492-9982-7a6c706d158a" />
